core/src/main/java/org/apache/james/core/Username.java

```diff
@@ -84,6 +86,8 @@ private Username(String localPart, Optional<Domain> domainPart) {
         Preconditions.checkNotNull(localPart);
         Preconditions.checkArgument(!localPart.isEmpty(), "username should not be empty");
         Preconditions.checkArgument(!localPart.contains("@"), "username can not contain domain delimiter");
+        Preconditions.checkArgument(!isContainInvalidCharacters(localPart),
+            "localPart should not contain consecutively dots");
```

Comment:
The ticket is about rejecting `..` in mailbox names, not in username.

Reply:
We don't allow users to be created anymore with invalid characters, however we still allow to read the ones already created, for backward-compatibility with potential James prod running instances within the community having users with such (we don't want their users screaming for not being able to access their mails anymore^^). Please remove this commit